ngx-fudis/projects/ngx-fudis/src/lib/components/dialog/ngMaterial-theme.scss

```diff
@@ -113,9 +121,10 @@
   @include colorMixins.bg-color('white');
 
   &.cdk-overlay-backdrop.cdk-overlay-backdrop-showing {
+    top: -50vh;
```

Comment:
Not sure what this does/fixes, if we have increased the `height` from `300vh` to `350vh` and then added `top: -50vh`?

Reply:
This is a hack that centres the viewport, so that the material dialog's background overlay won't overflow on Mobile-Chrome-Big-Landscape view